Makefile

```diff
@@ -76,7 +76,7 @@ clean: chart-clean
 regenerate-charts: chart-clean compile-chart
 
 regenerate-nix:
-	nix run -f . regenerateNixLockfiles
+	nix run --extra-experimental-features flakes -f . regenerateNixLockfiles
```

Comment:
Just noting here for any future reader that this was added so that non-nix users don't have to edit their nix.conf.

Reply:
Added in 33c9d89